rfc8446.xml

```diff
@@ -1455,7 +1348,7 @@ directly to these.</t>
 
 <t>Implementations MUST NOT send extension responses
 if the remote endpoint did not send the corresponding extension requests,
-with the exception of the "cookie" extension in a HelloRetryRequest.
+with the exception of the "cookie" extension in the HelloRetryRequest.
```

Comment:
I'm failing to remember why we changed this to "the", and wonder if it would read better as just "in HelloRetryRequest" in the sense of the abstract container.